daemon/algod/api/generate.sh

```diff
@@ -2,14 +2,21 @@
 set -e
 
 rootdir=`dirname $0`
-pushd $rootdir
+pushd ${rootdir}
+
+cd ${rootdir}/../../..
+$(GO111MODULE=on go get -u "github.com/algorand/oapi-codegen/...@v1.3.5-algorand2")
+
+popd
 
 # Convert v2 to v3
 curl -s -X POST "https://converter.swagger.io/api/convert" -H "accept: application/json" -H "Content-Type: application/json" -d @./algod.oas2.json  -o 3.json
 cat 3.json | json_pp > algod.oas3.yml
 rm 3.json
-sed -i 's/\*.\*/application\/json/g' algod.oas3.yml
+sed -i '' 's/\*.\*/application\/json/g' algod.oas3.yml
```

Comment:
This breaks in on Linux, does `sed --posix -i 's/\*.\*/application\/json/g' algod.oas3.yml` work on the mac?

Alternatively we might need something like this:
```
if [[ "$OSTYPE" == "darwin"* ]]; then
  sed -i '' 's/\*.\*/application\/json/g' algod.oas3.yml
else
  sed -i 's/\*.\*/application\/json/g' algod.oas3.yml
fi
```

Reply:
isn't this obsolete anyway? I don't think there are any `*.*` to be replaced in the current files.

Reply:
I thought I had them fully defined in the oas2 file, but they were getting changed to `*.*` after the conversion. I haven't checked this since adding the fix originally.

Reply:
Since I can't see *.* in the file, I'm going to comment out this line for now.